### DISCUSSION TITLE
Huge amount of Disk space for local instance

### DISCUSSION THREAD

**Discussion by iizem** [rsxdalv/TTS-WebUI#272] [General]
After a successful local installation, even for a simple test the generation takes an eternity and cache attends 35 Gb while the installation folder is more than 70 Gbs so a total of 115 Gbs without successfully generating a simple 2 lines of text to Audio.
highly recommand to use the Colab version rather than the local instance.

![Screenshot 2024-01-31 201200](https://github.com/rsxdalv/tts-generation-webui/assets/130265776/a031b987-db2a-4124-9e19-96fd4f7acea7)

**Comment by rsxdalv** [comment]
At this point in time colab is a good option for sure.
100gb is more than I've seen before.
May I ask what is the size of the separate folders: installer_files and tts-generation-webui?
I hope the cache isn't just this project, although models like Audiocraft's stereo large Musicgen are just massive.

**Reply by rsxdalv** [reply]
For reference, it should be something like this, potentially even less. Of course, the HF models cache will contain more models once you use them:

![image](https://github.com/rsxdalv/tts-generation-webui/assets/6757283/f38212d6-c456-44c7-bfa7-157b08369d4f)

**Comment by rsxdalv** [comment]
I will close the discussion for now but anyone can feel free to open it with more questions or details. I haven't seen much space-saving in the audio space, so it's possible that with some pressure model weights could become smaller, probably audiocraft being a big opportunity to save space.
